symfony/http-kernel/CVE-2015-4050.yaml

```diff
@@ -14,4 +14,7 @@ branches:
     2.6.x:
         time:     2015-05-26 23:55:51
         versions: [>=2.6.0,<2.6.8]
+    2.7.x:
+        time:     2015-05-26 23:55:51
+        versions: [>=2.7.0-dev,<2.7.0]
```

Comment:
@cebe no. You are confusing ``2.7.0-dev`` (dev snapshot tag, so before alphas) and ``2.7.x-dev`` which is the branch version.

And btw, this is the reason why I don't like the possibility to omit the ``.x`` part when specifying branch aliases (having composer add it silently) as it means ``2.7-dev`` has a different meaning only in this context